src/wallet.js

```diff
@@ -1194,18 +1194,34 @@ const wallet = {
 
   /*
    * Get output of a change of a transaction
+   * If changeAddress is not passed we get the next address from the wallet to use
    *
    * @param {number} value Amount of the change output
    * @param {number} tokenData Token index of the output
+   * @param {String} changeAddress Optional parameter with address for the change output
+   * @param {Object} options Options parameters
+   *  {
+   *   'address': address of the change output
+   *  }
    *
    * @return {Object} {'address': string, 'value': number, 'tokenData': number, 'isChange': true}
    *
    * @memberof Wallet
    * @inner
    */
-  getOutputChange(value, tokenData) {
-    const address = this.getAddressToUse();
-    return {'address': address, 'value': value, 'tokenData': tokenData, 'isChange': true};
+  getOutputChange(value, tokenData, options = { address: null }) {
+    const { address } = options;
+    if (address) {
+      if (!transaction.isAddressValid(address)) {
+        throw new AddressError('Change address is invalid.');
+      }
+      if (!this.isAddressMine(address)) {
+        throw new AddressError('Change address is not from loaded wallet.');
```

Comment:
Minor thing: The library doesn't know there might be multiple wallets. I'd change the error message to "Change address is not from this wallet.".

Reply:
I agree. Updated